melodic/distribution.yaml

```diff
@@ -2017,6 +2017,12 @@ repositories:
       url: https://github.com/clearpathrobotics/cpr_multimaster_tools.git
       version: kinetic-devel
     status: maintained
+  cpu_temperature_diagnostics:
+    source:
+      type: git
+      url: https://github.com/yotabits/cpu_temperature_diagnostics.git
```

Comment:
why is the release being done from a fork?

Reply:
Because the release is happening on my own initiative. Original repo organization does not have time to allocate to make and maintain a release.

Reply:
This is adding a package as a source entry, so there doesn't seem to be any release associated.
The upstream repo also seems to be responsive to your patches.

If you're planning to start making releases, you only need to have write access to the release repo, you don't need to be using a fork of the source repo.
Of course, if the upstream maintainer is not responsive we could update the source repo to be your fork.

---

@clalancette  do we have any policy related to when to forks?

Reply:
@ivanpauno @clalancette thank you for the feedback.
The original repo is a repository part of a company that does not need to release the package and will not allocate time to do it or maintain it.
Currently, I have write permission to this repository as I'm part of this company, if I happen to not be part of it anymore I will lose all my rights.
Consequently, I think it's better to use a fork I have control of instead of the original repository that can change independently of my will.
I'm the author of this package.

Reply:
Considering that you're the author (which I didn't notice before) and you prefer to add the fork as an upstream repo that sounds fair enough to me.
Though I agree with @clalancette that this is a bit confusing.